test/integration/spec/connect.js

```diff
@@ -453,7 +453,14 @@ describe('connect', function() {
         assert(pc.remoteDescription.sdp);
         return getMediaSections(pc.remoteDescription.sdp, 'audio');
       }).forEach(section => {
-        assert(!/b=(AS|TIAS):([0-9]+)/.test(section));
+        const codecMap = createCodecMap(section);
+        const codecIds = getCodecIds(section);
```

Comment:
I thought we started referring to these as payload types everywhere?

Reply:
@markandrus In the source, yes. I can change them in this test as well.

Reply:
If you don't mind, that would be nice 👍